setup.py

```diff
@@ -1,98 +1,16 @@
-plugin_license = "None"
-
-# Any additional requirements besides OctoPrint should be listed here
-plugin_requires = []
-
-### --------------------------------------------------------------------------------------------------------------------
-### More advanced options that you usually shouldn't have to touch follow after this point
-### --------------------------------------------------------------------------------------------------------------------
-
-# Additional package data to install for this plugin. The subfolders "templates", "static" and "translations" will
-# already be installed automatically if they exist.
-plugin_additional_data = []
-
-# Any additional python packages you need to install with your plugin that are not contained in <plugin_package>.*
-plugin_additional_packages = []
-
-# Any python packages within <plugin_package>.* you do NOT want to install with your plugin
-plugin_ignored_packages = []
-
-# Additional parameters for the call to setuptools.setup. If your plugin wants to register additional entry points,
-# define dependency links or other things like that, this is the place to go. Will be merged recursively with the
-# default setup parameters as provided by octoprint_setuptools.create_plugin_setup_parameters using
-# octoprint.util.dict_merge.
-#
-# Example:
-#     plugin_requires = ["someDependency==dev"]
-#     additional_setup_parameters = {"dependency_links": ["https://github.com/someUser/someRepo/archive/master.zip#egg=someDependency-dev"]}
-additional_setup_parameters = {}
-
-########################################################################################################################
-
-from setuptools import setup
-
-try:
-	import octoprint_setuptools
-except:
-	print("Could not import OctoPrint's setuptools, are you sure you are running that under "
-	      "the same python installation that OctoPrint is installed under?")
-	import sys
-	sys.exit(-1)
-
-setup_parameters = octoprint_setuptools.create_plugin_setup_parameters(
-	identifier=plugin_identifier,
-	package=plugin_package,
-	name=plugin_name,
-	version=plugin_version,
-	description=plugin_description,
-	author=plugin_author,
-	mail=plugin_author_email,
-	url=plugin_url,
-	license=plugin_license,
-	requires=plugin_requires,
-	additional_packages=plugin_additional_packages,
-	ignored_packages=plugin_ignored_packages,
-	additional_data=plugin_additional_data
-)
-
-if len(additional_setup_parameters):
-	from octoprint.util import dict_merge
-	setup_parameters = dict_merge(setup_parameters, additional_setup_parameters)
-
-setup(**setup_parameters)
+from distutils.core import setup
+
+exec(open('octoprint_mrbeamdoc/__version.py').read())
+
+setup(name='MrBeamDoc',
+      version=__version__,
+      description="""Provides documentation for Mr Beam Plugin""",
+      author='Mr Beam',
+      author_email='dev@mr-beam.org',
+      url='https://github.com/mrbeam/MrBeamDoc',
+      packages=['octoprint_mrbeamdoc'],
```

Comment:
maybe rename the package to only mbreamdoc,  but you should check if the update still works

Reply:
lets bring this up on our next stand up and see what the team thinks

Reply:
ok

Reply:
As discussed, we will keep the same name